controllers/infinispan_controller.go

```diff
@@ -290,6 +290,37 @@ func (reconciler *InfinispanReconciler) Reconcile(ctx context.Context, ctrlReque
 		}
 	}
 
+	if infinispan.HasSites() {
+		reqLogger.Info("Creating the Cross-Site Deployment (Gossip Router)")
+		tunnel := r.GetGossipRouterDeployment(infinispan)
+		_, err = controllerutil.CreateOrUpdate(r.ctx, r.Client, tunnel, func() error {
+			return controllerutil.SetControllerReference(r.infinispan, tunnel, r.scheme)
+		})
+		if err != nil {
+			reqLogger.Error(err, "Failed to configure Cross-Site Deployment")
+			return reconcile.Result{}, err
+		}
+		reqLogger.Info("Cross-Site Deployment created")
```

Comment:
@pruivo you will always print  "Cross-Site Deployment created" on each reconcile loop whenever it created or updated
second parameter of the `controllerutil.CreateOrUpdate` returns performed operation